### PR TITLE
Add Sympa FCGI role based on systemd template unit files.

### PR DESCRIPTION
**Pull request by racke**
This is a proof of concept. I'm going to add Nginx provisioning.

### CODE REVIEW

**Comment by dverdin** [comment]
Interesting. Is the idea behind to make the use of Sympa web tools easily optional?

**Comment by racke** [comment]
That will be eventually the implementation for #58.

**Comment by dverdin** [comment]
Cool. That looks conflicting with PR #56 though.
I think we should validate it to have something working and leave us time to go forward with this PR.

**Comment by racke** [comment]
> Cool. That looks conflicting with PR #56 though.
> I think we should validate it to have something working and leave us time to go forward with this PR.

You need to fix handlers + systemd in #56 first.

**Comment by racke** [comment]
The Nginx part is quite simple, we need to add a config file like that:

```
upstream sympa_backend {
    server unix:/run/sympa/sympa1.sock;
    server unix:/run/sympa/sympa2.sock;
    server unix:/run/sympa/sympa3.sock;
    server unix:/run/sympa/sympa4.sock;
    server unix:/run/sympa/sympa5.sock;
}
```

And in the virtual hosts we are going to adjust `fast_cgi_pass`:

```
   fastcgi_pass sympa_backend;
```

File: roles/sympa-fcgi/templates/service.j2 [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description = {{ unit_description }} (service instance %i)
+After = syslog.target
+BindTo = sympa.service
```
Comment (ikedas) [inline roles/sympa-fcgi/templates/service.j2]: On upstream link between sympa and web services has been weaken, like
``` code
After = syslog.target sympa.service
```
cf. sympa-community/sympa#525

Comment (racke) [reply roles/sympa-fcgi/templates/service.j2]: Fixed through 7b05561. Thanks @ikedas !

**Comment by racke** [comment]
Repository is obsolete.

**Pull request closed without merging**